test/langtools/tools/javac/reflect/ForLoopTest.java

```diff
@@ -36,36 +36,36 @@
 public class ForLoopTest {
     @CodeReflection
     @IR("""
-            func @"test1" (%0 : ForLoopTest, %1 : java.util.List<java.util.List<java.lang.String>>)void -> {
-              %2 : Var<java.util.List<java.util.List<java.lang.String>>> = var %1 @"ll";
-              java.enhancedFor
-                  ^expr()java.util.List<java.util.List<java.lang.String>> -> {
-                      %3 : java.util.List<java.util.List<java.lang.String>> = var.load %2;
-                      yield %3;
-                  }
-                  ^def(%4 : java.util.List<java.lang.String>)Var<java.util.List<java.lang.String>> -> {
-                      %5 : Var<java.util.List<java.lang.String>> = var %4 @"l";
-                      yield %5;
-                  }
-                  ^body(%6 : Var<java.util.List<java.lang.String>>)void -> {
-                      java.enhancedFor
-                          ^expr()java.util.List<java.lang.String> -> {
-                              %7 : java.util.List<java.lang.String> = var.load %6;
-                              yield %7;
-                          }
-                          ^def(%8 : java.lang.String)Var<java.lang.String> -> {
-                              %9 : Var<java.lang.String> = var %8 @"s";
-                              yield %9;
-                          }
-                          ^body(%10 : Var<java.lang.String>)void -> {
-                              %11 : java.io.PrintStream = field.load @"java.lang.System::out()java.io.PrintStream";
-                              %12 : java.lang.String = var.load %10;
-                              invoke %11 %12 @"java.io.PrintStream::println(java.lang.String)void";
-                              java.continue;
-                          };
-                      java.continue;
-                  };
-              return;
+            func @"test1" (%0 : java.type:"ForLoopTest", %1 : java.type:"java.util.List<java.util.List<java.lang.String>>")java.type:"void" -> {
+                %2 : Var<java.type:"java.util.List<java.util.List<java.lang.String>>"> = var %1 @"ll";
+                java.enhancedFor
+                    ()java.type:"java.util.List<java.util.List<java.lang.String>>" -> {
```

Comment:
Here and elsewhere -- it seems like block names like `^expr` or `^def` are no longer generated. But this PR doesn't change that (in fact, I tried creating the model for a simple for loop without these changes, and I still don't see the body names).

I believe this has to do with the fact that e.g. for ops are composed of _bodies_, and each body in the op has a single block (thus, it is the entry block for that body). And, entry block names are skipped.

The test was probably still working fine (despite the mismatch) because we parse the IR string, then generate the textual form again before comparing it -- this will drop the entry block names -- meaning the "expected" IR will match the one generated by javac.